docs/resource_creation_guide.md

```diff
@@ -77,11 +77,11 @@ Consider the following:
 - Update variables and Terraform outputs as needed.
 
 ```
-resource_group = attribute('resource_group',  default: nil)
-subnet         = attribute('subnet_name',     default: nil)
-tags           = attribute('subnet_tags',     default: nil)
-id             = attribute('subnet_id',       default: nil)
-location       = attribute('subnet_location', default: nil)
+resource_group = attribute('resource_group',  value: nil)
```

Comment:
If making the `default` -> `value` change might be best also to update `attribute` to `input` at the same time.

Reply:
Done!